### PR TITLE
Add missing typing hints for mypy checks

### PR DESCRIPTION
**Pull request by jumcorredorro**
## Description
Add missing typing hints for mypy checks. 
There are some ineludible ignores. Mypy is unable to check this lines right.

## Changes include
- [X] Bugfix (non-breaking change that solves an issue, bump the third digit of the version)
- [ ] New feature (non-breaking change that adds functionality, bump the second digit of the version)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality, bump the first digit of the version)
- [ ] Documentation update (Don't bump the version of the project)

### CODE REVIEW

File: drip/models.py [hunk 1]
```diff
@@ -186,17 +190,17 @@ def annotated_field_name(self) -> str:
 
         return field_name
 
-    def apply_any_annotation(self, qs: Optional[QuerySet]) -> Optional[QuerySet]:
+    def apply_any_annotation(self, qs: QuerySet) -> QuerySet:
         """
         Returns qs annotated with Count over this field's name.
         """
-        if self.field_name.endswith("__count") and qs is not None:
```
Comment (brunomichetti) [inline drip/models.py:193]: Why did you remove `and qs is not None`?

File: drip/admin.py [hunk 2]
```diff
@@ -34,10 +37,16 @@ class DripAdmin(admin.ModelAdmin):
...
     users_fields: Union[str, List[str]] = []
 
-    def av(self, view):
+    def av(self, view: Callable) -> Callable:
         return self.admin_site.admin_view(view)
 
-    def timeline(self, request, drip_id, into_past, into_future):
+    def timeline(
+        self,
+        request: WSGIRequest,
```
Comment (malexandroff1) [inline drip/admin.py:45]: can you use  HttpRequest instead of WSGIRequest or they are different?

File: drip/admin.py [hunk 3]
```diff
@@ -79,36 +90,42 @@ def get_mime_html(self, drip, user):
         mime = "text/plain"
         return html, mime
 
-    def view_drip_email(self, request, drip_id, into_past, into_future, user_id):
-
+    def view_drip_email(
+        self,
+        request: WSGIRequest,
```
Comment (malexandroff1) [inline drip/admin.py:95]: same here

File: drip/models.py [hunk 4]
```diff
@@ -43,11 +46,12 @@ class Meta:
         abstract = True
 
     @property
-    def drip(self):
+    def drip(self) -> "DripBase":
```
Comment (malexandroff1) [inline drip/models.py:49]: This can be moved to DripBase as a class method. For instance, DripBase.create_from_abstract_drip(drip: AbstractDrip) -> DripBase, so you can avoid the circular import

**Review by malexandroff1 — APPROVED** [review]
LG2M

File: drip/models.py (hunk 1 above)
Comment (jumcorredorro) [reply drip/models.py:193]: I added it in another PR, but now it is not needed because this function should not get a None QuerySet.

File: drip/admin.py (hunk 2 above)
Comment (jumcorredorro) [reply drip/admin.py:45]: This two classes are "similar" but checking the types here, this functions are getting a `WSGIRequest`.

File: drip/models.py (hunk 4 above)
Comment (jumcorredorro) [reply drip/models.py:49]: I will change this in the future. Thanks!

**Review by brunomichetti — APPROVED** [review]
